arrows/vtk/applets/CMakeLists.txt

```diff
@@ -43,3 +43,7 @@ kwiver_add_plugin( kwiver_algo_vtk_applets
                kwiversys
                ${VTK_targets}
   )
+
+target_link_libraries( kwiver_algo_vtk_applets
+  PRIVATE              kwiver_algo_pdal
+  )
```

Comment:
I think you can put this library right in the list above in `kwiver_add_plugin`

Reply:
Actually, we may want to do it this way but wrap this in a conditional `if( KWIVER_ENABLE_PDAL )`.  We should move all that conditional logic with cmakedefines and ifdefs to here from the PDAL arrow.  In the PDAL itself we don't need that logic because either we build the whole arrow or we don't depending on whether PDAL is enabled.  However, we want to build the VTK arrow whether or not PDAL is enabled and only add these PDAL dependencies if PDAL is also enabled.  Does that make sense?

Reply:
Makes sense to me. In my implementation I left the `pdal_config.h.in` file in the pdal arrow. Should I move that into the vtk applets as well?